templates/tower-project.j2

```diff
@@ -351,8 +351,9 @@ Resources:
             - Id: SynapseCORSRule
               AllowedHeaders: ["*"]
               AllowedOrigins: ["*"]
-              AllowedMethods: [GET, POST, PUT, HEAD]
+              AllowedMethods: ["GET", "POST", "PUT", "HEAD", "DELETE"]
```

Comment:
Example in [Seqera Enterprise S3 CORS configuration docs](https://docs.seqera.io/platform-cloud/data/data-explorer) indicates that "AllowedOrigins" should be a specific endpoint (i.e. `["https://<your-seqera-instance.url>"]`).  I'm thinking that existing `*` configuration should work as well?

Reply:
Yup, got confirmation from Seqera saying that `*` should work therefore I’m inclined to keep it as is.